notebooks/amazon_forest_notebook.py

```diff
@@ -212,39 +210,40 @@
 from tensorflow.contrib.keras.api.keras.callbacks import ModelCheckpoint
 
 filepath="weights.best.hdf5"
-checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True, mode='max')
+checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True)
 
 # <markdowncell>
 
-# ## Create the neural network definition
+# ## Define and Train model
+
+# <markdowncell>
+
+# Here we define the model and begin training. 
+# 
+# Before starting the training process, you should first set a learning rate annealing optimization schedule by choosing a series of learning rates (learn_rates) with corresponding number of epochs for each (epochs_arr).
```

Comment:
In markdown code variables are usually contained in ` quotes.

...schedule by choosing a series of learning rates (`learn_rates`) with...

...number of epochs for each (`epochs_arr`).